README.md

```diff
@@ -1,2 +1,54 @@
-# opensearch-dashboards-snap
-`snap` package providing Opensearch Dashboards
+# OpenSearch Dasboards Snap
+
+[//]: # (<h1 align="center">)
+[//]: # (  <a href="https://opensearch.org/">)
+[//]: # (    <img src="https://opensearch.org/assets/brand/PNG/Logo/opensearch_logo_default.png" alt="OpenSearch" />)
+[//]: # (  </a>)
+[//]: # (  <br />)
+[//]: # (</h1>)
+
+This is the snap package for [OpenSearch Dashboards](https://opensearch.org/docs/latest/dashboards/), a
+community-driven, Apache 2.0-licensed user interface that lets you visualize your OpenSearch data, together
+with running and scaling your OpenSearch clusters.
+
+
+
+### Installation:
+[![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/opensearch-dashboards)
+
+or:
+```
+sudo snap install opensearch-dashboards --channel=2/candidate
+```
+
+### Starting OpenSearch Dashboards:
+
+#### Starting OpenSearch:
+```
+sudo snap start opensearch-dashboards
+```
+Additional parameters may be set in the shape of shell environment variables, such as
+
```

Comment:
can we add a full command here? i.e:
`sudo snap set opensearch-dashboards port=1234`